chpl/chpl-service/src/main/resources/log4j2-app.xml

```diff
@@ -10,7 +10,7 @@
         <RollingFile name="FILE" fileName="${logDir}/${appName}.log"
             filePattern="${logDir}/${appName}-%d{yyyyMMdd}.log">
             <PatternLayout>
-                <Pattern>%d %p %C %m%n</Pattern>
+                <Pattern>%d{ISO8601} %-5p (%t) [%c{1}(%M:%L)] %m%n</Pattern>
```

Comment:
Did you intend to change the app logging? It's supposed to go to a file on the other systems.